crates/core/c8y_api/src/http_proxy.rs

```diff
@@ -266,6 +272,25 @@ impl JwtAuthHttpProxy {
         )
     }
 
+    fn create_event(
```

Comment:
I'm a bit surprised to see this function tagged by the diff as being *added* since it's already defined although in a different manner.

Reply:
This generic function was created to enable reuse by any plugin that needs to create events in future. The log plugin could also use this function in future to create log events of different log types.